lib/v2/xaufe/jiaowu.js

```diff
@@ -0,0 +1,77 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const cheerio = require('cheerio');
+
+// Yep http is bad, but I had no choice :(
+const rootMeta = {
+    url: 'http://jiaowu.xaufe.edu.cn/',
+    title: '西安财经大学 教务处（招生办公室）',
+};
+
+const categories = {
+    tzgg: {
+        title: '通知公告',
+        url: 'index/tzgg.htm',
+    },
+};
+
+module.exports = async (ctx) => {
+    const pCategory = ctx.params.category;
+    const category = categories[pCategory] || categories.tzgg;
+
+    const response = (
+        await got({
+            method: 'get',
+            url: rootMeta.url + category.url,
+        })
+    ).body;
+
+    const $ = cheerio.load(response);
+
+    const data = $('ul', '.main_conRCb')
+        .children('li')
+        .slice(0, 20)
+        .map((_, item) => {
+            item = $(item);
+            const pubDate = item.children('span').text();
+            const title = item.find('a em').text();
+            const link = item
+                .children('a')
+                .attr('href')
+                .replace(/\.\.\//g, rootMeta.url);
+            return {
+                pubDate: parseDate(pubDate),
+                title,
+                link,
+            };
+        })
+        .get();
+
+    ctx.state.data = {
+        title: `${category.title}-${rootMeta.title}`,
```

Comment:
I believe you can obtain the same value using $('head title').text()

Reply:
Yep, but there's some other website may apply here in the future, they may not equal to the HTML title xD